doc/.pre_processing.py

```diff
@@ -23,10 +23,16 @@ class Colors:
     ERROR = '\033[91m'
     END = '\033[0m'
     
-if not os.path.isfile('./conf.py'):
+if not os.path.isfile('./.conf.py'):
   print("Could not find Sphinx's configuration file. Make sure to start this script within Mantella's documentation root path.")
   exit(1)
 
+# Setting the versions number for conf.py
+searchresult = re.search(r'MANTELLA_VERSION_MAJOR[ ]+(\d+)\n.*[ ](\d+)', open(''.join(glob.glob('../include/mantella[0-9]')), mode='r', encoding='utf-8').read())
+actualVersion = searchresult.group(1) + '.' + searchresult.group(2)
+conf_file = open('./.conf.py', mode='r', encoding='utf-8').read()
+open('./conf.py', mode='w', encoding='utf-8').write(re.sub(r'(project = .*)', '\\1\\nversion = u\'' + actualVersion + '\'' , conf_file, 0))
```

Comment:
Why do we need to do this? As `conf.py` is a python script in it self, couldn't `conf.py` also do it by itself?